pkg/monitoring/vmstat/hyperv/provider_windows.go

```diff
@@ -0,0 +1,70 @@
+// +build windows
+
+package hyperv
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cloudradar-monitoring/cagent/perfcounters"
+	"github.com/cloudradar-monitoring/cagent/pkg/monitoring"
+	"github.com/cloudradar-monitoring/cagent/pkg/monitoring/vmstat"
+	"github.com/cloudradar-monitoring/cagent/pkg/wmi"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	prov := newImpl()
+
+	err := vmstat.RegisterVMProvider(prov)
+	if err != nil {
+		panic(err.Error())
+	}
+}
+
+type impl struct {
+	watcher *perfcounters.WinPerfCountersWatcher
+}
+
+var _ vmstat.Provider = (*impl)(nil)
+
+func newImpl() vmstat.Provider {
+	return &impl{
+		watcher: monitoring.GetWatcher(),
+	}
+}
+
+func (im *impl) Run() error {
+	interval := time.Second * 1
+
+	err := im.watcher.StartContinuousQuery(hypervPath, interval)
+	if err != nil {
+		log.Printf("Failed to StartQuery: %s", err)
```

Comment:
In case we return the `err` we should log the error in the level above.
If an error is logged it's basically handled.

Reply:
done